lib/archethic.ex

```diff
@@ -69,16 +56,32 @@ defmodule Archethic do
         welcome_node_key \\ Crypto.first_node_public_key()
       ) do
     if P2P.authorized_and_available_node?() do
-      do_send_transaction(tx, welcome_node_key)
+      case P2P.verify_synchronization() do
+        {:ok, :node_in_sync} ->
+          do_send_transaction(tx, welcome_node_key)
+
+        {:error, :node_out_of_sync} ->
```

Comment:
I think we should not start a repair if the verify synchronization did not retrieve the last_address. Starting a resync makes no sensse as we don't know if we are sync or not. So the verify_synchronization could just return :ok, {:error, last_address} or :error.
In case of just error we forward the tx without starting a repair.